commit 94abd3ca2c51facd2571e56bad2e5fee8a889c64
Author: Herbert Xu <herbert@gondor.apana.org.au>
Date:   Wed Aug 2 16:40:47 2017 +0800

    crypto: ixp4xx - Fix error handling path in 'aead_perform()'
    
    commit 28389575a8cf933a5f3c378556b9f4d3cce0efd2 upstream.
    
    In commit 0f987e25cb8a, the source processing has been moved in front of
    the destination processing, but the error handling path has not been
    modified accordingly.
    Free resources in the correct order to avoid some leaks.
    
    Fixes: 0f987e25cb8a ("crypto: ixp4xx - Fix false lastlen uninitialised warning")
    Reported-by: Christophe JAILLET <christophe.jaillet@wanadoo.fr>
    Signed-off-by: Herbert Xu <herbert@gondor.apana.org.au>
    Reviewed-by: Arnd Bergmann <arnd@arndb.de>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1635_94abd3ca2c51facd2571e56bad2e5fee8a889c64.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1635_94abd3ca2c51facd2571e56bad2e5fee8a889c64.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
